bref/symfony-bridge/0.1/serverless.yaml

```
@@ -0,0 +1,46 @@
+# Read the documentation at https://www.serverless.com/framework/docs/providers/aws/guide/serverless.yml/
+service: symfony
+
+provider:
+    name: aws
+    # The AWS region in which to deploy (us-east-1 is the default)
+    region: us-east-1
+    # The stage of the application, e.g. dev, production, staging… ('dev' is the default)
+    stage: dev
+    runtime: provided.al2
+    environment:
+        # Symfony environment variables
+        APP_ENV: prod
+
+plugins:
+    - ./vendor/bref/bref
+
+functions:
+    # This function run the Symfony application
+    website:
```

```suggestion
    web:
```

Inspired from the Laravel config (https://github.com/brefphp/laravel-bridge/blob/master/config/serverless.yml#L22), that leaves room for APIs.

(BTW I realize that you copied the official Bref documentation :) I'm just taking the opportunity to improve what can be improved)